Modules/_ssl.c

```diff
@@ -5925,8 +5925,7 @@ sslmodule_init_constants(PyObject *m)
 #define addbool(m, key, value) \
     do { \
         PyObject *bool_obj = (value) ? Py_True : Py_False; \
-        Py_INCREF(bool_obj); \
-        PyModule_AddObject((m), (key), bool_obj); \
+        PyModule_AddObjectRef((m), (key), bool_obj); \
```

Comment:
Isn't this still a leak because we're not checking the return value?

Reply:
It's no longer a leak.

The thing that makes `PyModule_AddObject` tricky cause leaks is that you have to decref in the failure case, but not in the success case (because it steals the ref). With `PyModule_AddObjectRef` you can treat the failure and success cases the same in terms of what they do to ref counts.

But yes, in general we should probably be propagating failures here, and the other cases below you commented on. I didn't do it in this PR because it's a slightly separate change and there are many other places in this module where we don't check return values.

Reply:
Should I fix just the ones I'm touching in this PR or fix all of them?

Reply:
Thanks for the explanation! I feel it'd be fine to also fix missing error checks in this PR, since the change remains localized and you're fixing a very similar problem to what the PR already fixes.